oracle/blockeligibilityvalidator.go

```diff
@@ -13,24 +13,26 @@ import (
 type VRFValidationFunction func(message, signature, publicKey []byte) (bool, error)
 
 type BlockEligibilityValidator struct {
-	committeeSize  uint32
-	layersPerEpoch uint16
-	activationDb   ActivationDb
-	beaconProvider *EpochBeaconProvider
-	validateVRF    VRFValidationFunction
-	log            log.Log
+	committeeSize        uint32
+	genesisActiveSetSize uint32
+	layersPerEpoch       uint16
+	activationDb         ActivationDb
+	beaconProvider       *EpochBeaconProvider
+	validateVRF          VRFValidationFunction
+	log                  log.Log
 }
 
-func NewBlockEligibilityValidator(committeeSize int32, layersPerEpoch uint16, activationDb ActivationDb,
+func NewBlockEligibilityValidator(committeeSize uint32, genesisActiveSetSize uint32, layersPerEpoch uint16, activationDb ActivationDb,
```

Comment:
not critical, but you can omit the first `uint32` here

Reply:
Removed